src/lib/telemetry/TelemetryCollector.ts

```diff
@@ -1,10 +1,17 @@
+import { randomBytes } from 'crypto'
+import { ensureFile, writeJson } from 'fs-extra'
 import { spawn } from 'child_process'
 import { join } from 'path'
+
 import * as pkgJson from '../../../package.json'
 import { ErrorReport } from '../error/ErrorReport'
 import { ITelemetryLocalStore, TelemetryLocalStore } from './TelemetryStore'
+import { configDir } from '../../conf'
+import { logger } from '../../clients'
```

Comment:
Oops, that is not the desired logger, it should be:
```
import logger from '../../logger'
```
That's one of the reasons I don't like `default exports` haha, normally vscode won't show that there's a logger in this file